### PR TITLE
docs: add lazy.nvim install description

### PR DESCRIPTION
**Pull request by nolleh**
add install description for lazy.nvim

description for lazy.nvim: 
- https://github.com/folke/lazy.nvim
- https://www.lazyvim.org/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,7 +70,31 @@ Plugin 'iamcco/markdown-preview.nvim'
...
+Or with [lazy.nvim](https://github.com/folke/lazy.nvim):
 
+Add this in your `init.lua or plugins.lua`
+
+```lua
+-- install without yarn or npm
+{
+    "iamcco/markdown-preview.nvim",
+    cmd = { "MarkdownPreviewToggle" },
+    ft = { "markdown" }
```
Comment (DerpGusta) [inline README.md:82]: just need to add a comma here.

Comment (nolleh) [reply README.md]: oh, thanx, I've just added the comma you menthioned.

**Comment by gabyx** [comment]
See #587 , thats the correct one

**Comment by nolleh** [comment]
@gabyx
I failed to find out reply menu for your comment.

and could you kindly tell me which point is incorrect?

it's looks like personal favor,,, ^^;
tnx for opinon.

**Comment by nolleh** [comment]
@gabyx 
but, after seeing your PR and seeing mkdp.vim file (the plugin command definition),
It look's like other people may want to additional comment.  user can use any of these command. 
(to handle below command.)
```
  noremap <buffer> <silent> <Plug>MarkdownPreview :MarkdownPreview<CR>
  noremap <buffer> <silent> <Plug>MarkdownPreviewStop :MarkdownPreviewStop<CR>
  nnoremap <buffer> <silent> <Plug>MarkdownPreviewToggle :MarkdownPreviewToggle<CR>
```

So applied your opinion. ( both of us PR **WAS** actually not fully handle user needs ^^;)
now fully handle, from now.

**Comment by V4G4X** [comment]
This is what I'm using:
```
{
        "iamcco/markdown-preview.nvim",
        cmd = { "MarkdownPreviewToggle" },
        ft = { "markdown" },
        build = function() vim.fn["mkdp#util#install"]() end,
}, -- Markdown Preview
```

From what I understand, other than restarting neovim, no other change is required.  

But when I do that and try `:MarkdownPreview` it does not open preview window. 

How do I fix this?  
Where can I find logs?

**Comment by V4G4X** [comment]
This is what I'm using:
```
{
        "iamcco/markdown-preview.nvim",
        cmd = { "MarkdownPreviewToggle" },
        ft = { "markdown" },
        build = function() vim.fn["mkdp#util#install"]() end,
}, -- Markdown Preview
```

From what I understand, other than restarting neovim, no other change is required.  

But when I do that and try `:MarkdownPreview` it does not open preview window. 

How do I fix this?  
Where can I find logs?

**Comment by nolleh** [comment]
> This is what I'm using:
> 
> ```
> {
>         "iamcco/markdown-preview.nvim",
>         cmd = { "MarkdownPreviewToggle" },
>         ft = { "markdown" },
>         build = function() vim.fn["mkdp#util#install"]() end,
> }, -- Markdown Preview
> ```
> 
> From what I understand, other than restarting neovim, no other change is required.
> 
> But when I do that and try `:MarkdownPreview` it does not open preview window.
> 
> How do I fix this? Where can I find logs?

actually it is little far from PR's contents, but want it to help, 
I will reply :) 

1. in my envrionment, it also working by using `:MarkdownPreview` (look like there is some differences yours and mine in env.)
2. if you want to print log (with debug) in markdown-preview, follow the below. 

in `nvim(lua)`
```lua
-- the path any where you want to write
vim.env.NVIM_MKDP_LOG_FILE = "/Users/nolleh/tmp/mkdp.log"
vim.env.NVIM_MKDP_LOG_LEVEL = "debug"
```

in my env,  log remains like this. (after running `:MarkdownPreview`)

```log
2023-08-14 18:37:34 INFO (pid:43524) [app/server] - server run:  8672
2023-08-14 18:37:34 INFO (pid:43524) [app/server] - open page [default]:  http://localhost:8672/page/1
2023-08-14 18:37:34 INFO (pid:43524) [app/server] - refresh page:  1
2023-08-14 18:37:34 INFO (pid:43524) [app/server] - client connect:  mjrIp6OReWKVOwUTAAAA 1
```
hope this will help you.

**Pull request merged**